app/views/welcome/index.html.erb

```diff
@@ -111,6 +111,10 @@ body {font-family: "Lato", sans-serif}
         <%= link_to "Be The Change. Learn More", "/learn_more", class: "w3-btn w3-light-green w3-block" %>
       </div>
 
+      <div class="w3-panel">
+        <%= link_to "Garden Search", "/gardens", class: "w3-btn w3-blue-grey  w3-block" %>
+      </div>
+
```

Comment:
All this looks great!  Once we have the '/gardens' view in place, let's merge this in.

Reply:
@LHJE my latest PR (in review) adds garden index!